t/40-script_openqa-clone-custom-git-refspec.t

```diff
@@ -103,26 +102,24 @@ $expected_re = qr/${expected}/s;
 test_once $args, $expected_re, 'clone-job command with multiple URLs in PR and job URL';
 
 $ENV{curl_github} = qq{echo -e '{"head": {"label": "user:my/branch"}, "body": "Lorem ipsum"}'; true};
-my $needles = 'my/distri/products/sle/needles';
+my $needles = 'products/sle/needles';
 $ENV{curl_openqa}
   = qq{echo -e '{"TEST": "my_test", "CASEDIR": "my/distri", "PRODUCTDIR": "distri/products/sle", "NEEDLES_DIR": "$needles"}'; true};
```

Comment:
I wonder about this change. This variable simulates what openQA returns but only mentions the values we need. If we really don't need NEEDLES_DIR anymore then this should be removed as well then.

Reply:
I just left it there as a test case which NEEDLES_DIR is used. But I also believe that it doesnt help much. say the word and I will remove it